docker/puppetdb/docker-entrypoint.sh

```diff
@@ -1,24 +1,28 @@
-#!/bin/bash
+#!/bin/sh
 
 master_running() {
     status=$(curl --silent --fail --insecure "https://${PUPPETSERVER_HOSTNAME}:8140/status/v1/simple")
     test "$status" = "running"
 }
 
 PUPPETSERVER_HOSTNAME="${PUPPETSERVER_HOSTNAME:-puppet}"
-/opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
-/opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
+# /opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
+# /opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
 
 if [ ! -f "/etc/puppetlabs/puppet/ssl/certs/${HOSTNAME}.pem" ] && [ "$USE_PUPPETSERVER" = true ]; then
   # if this is our first run, run puppet agent to get certs in place
   while ! master_running; do
     sleep 1
   done
   set -e
-  /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
+  # /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
+  /ssl.sh
 fi
 if [ ! -d "/etc/puppetlabs/puppetdb/ssl" ] && [ "$USE_PUPPETSERVER" = true ]; then
-  /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  # /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  /ssl-setup.sh -f
 fi
 
-exec /opt/puppetlabs/server/bin/puppetdb "$@"
+# exec /opt/puppetlabs/server/bin/puppetdb "$@"
+exec java -cp /puppetdb.jar clojure.main -m puppetlabs.puppetdb.core "$@" \
```

Comment:
This needs a reference to `$JAVA_ARGS`, like:

```
exec java -cp /puppetdb.jar $JAVA_ARGS clojure.main -m puppetlabs.puppetdb.core "$@" \
    -c /etc/puppetlabs/puppetdb/conf.d/
```

Reply:
Added and verified here: https://github.com/puppetlabs/puppetdb/pull/2775/files#diff-86cc2c2467f4688ca87a833d8bf198b2R22